### ISSUE TITLE
Invalid plugin options for "gatsby-plugin-google-analytics"

### ISSUE THREAD

**Issue by Lukious** [JaeYeopHan/gatsby-starter-bee#227]
## Description
![image](https://user-images.githubusercontent.com/25575989/106703559-cdf6d800-662d-11eb-8a35-6134e41c71e0.png)

I tried 'npm start' as a readme guide but I got such an error.

To solve this problem I tried such as
1. downgrade webpack (npm install webpack-dev-server@~2.9.4) 
2. reinstalling npm
3. rm all stuff and re-installing (Node.js & NPM)
All these works were doesn't help me.

I think its caused by some google analytics version issue or deprecation error

*ps. my node version is 12.20.1 and my npm version is 6.14.10
thx

**Comment by freckie** [comment]
You must set the variable `ga` in gatsby-meta-config.js
(google analytics tracking id)

**Comment by Lukious** [comment]
@freckie 
Yes, I found it and typed it in, but according to README.md our build process should be like
1. Create a Gatsby site
2. Start developing
3. Add your content
4. Fix metadata
(I just reference its subtitles)

But, because of this error user should edit metadata first to set up this skin.
I thought we can solve this problem two-way
1. Edit README.md as like  Create a Gatsby site->Fix metadata->Start developing->Add your content
2. or, Just input '0'(any other value was possible) on our metadata setting

I thought method 2 is much better for our maintenance so I made PR like this https://github.com/JaeYeopHan/gatsby-starter-bee/pull/229

I will close this issue after my PR accept.

Thanks.
